db/migrate/20190122151538_create_new_risk_triggers.rb

```diff
@@ -0,0 +1,38 @@
+class CreateNewRiskTriggers < ActiveRecord::Migration
+  def change
+    # Create a new table
+    create_table :risk_triggers do |t|
+      t.string :name, null: false
+      t.string :matching_type, null: false
+      t.string :comment
+    end
+
+    # Add a reference field to the risk_trigger_conditions
+    add_reference :risk_trigger_conditions, :risk_trigger, index: true
+
+    # Create a default risk trigger
+    default_risk_trigger = RiskTrigger.create(
+      name: 'Default',
+      matching_type: 'all'
+    )
+
+    # Assign all the exisiting conditions to the default trigger
+    RiskTriggerCondition.update_all(risk_trigger_id: default_risk_trigger)
+
+    # Create a risk trigger conditions from the ones that were hard-coded in the RiskTrigger model
```

Comment:
Original logic (cells represent outcome of `risky?`:

 . | email = `google@lumendatabase.org` | email = something else
-|---------------------------------------|------------------------
Type = `Defamation` | `false`                  | ?
Type = something else | ?                 | ?

The ? means "other risk triggers are applied if they exist, so the outcome may or may not be risky".

This logic (I think; I'm finding it hard to follow with all the negations):

 . | email = `google@lumendatabase.org` | email = something else
-|---------------------------------------|------------------------
Type = `Defamation` | `false && false`, hence `false`  | `true && false`, hence `false`
Type = something else | `false && true`, hence `false` | `true && true`, hence `true`

Two conclusions:

1) This logic does not match the original and should be amended. 
2) Although I get why the negations are in there, this system might be extremely confusing to use and result in unintentional errors. Can we provide a less expressive system and still meet Adam's needs? (Possibly we want `any all none` as options on the RiskTrigger, rather than `negated` on the conditional?)